wyzesense2mqtt.py

```diff
@@ -93,34 +93,34 @@ def init_wysesense_dongle():
                         CONFIG['usb_dongle'] = "/dev/%s" % w
                         break
 
-    _LOGGER.info("Openning connection to dongle {0}".format(CONFIG['usb_dongle']))
-    wysesense_dongle = wyzesense.Open(CONFIG['usb_dongle'], on_event)
-    _LOGGER.debug("  MAC: {0}, VER: {1}, ENR: {2}".format(wysesense_dongle.MAC, wysesense_dongle.Version, wysesense_dongle.ENR))
+    LOGGER.info("Openning connection to dongle {0}".format(CONFIG['usb_dongle']))
+    WYSESENSE_DONGLE = wyzesense.Open(CONFIG['usb_dongle'], on_event)
+    LOGGER.debug("  MAC: {0}, VER: {1}, ENR: {2}".format(WYSESENSE_DONGLE.MAC, WYSESENSE_DONGLE.Version, WYSESENSE_DONGLE.ENR))
 
 # Initialize sensor configuration
-def init_sensors(sensors_config_file):
+def init_sensors():
     global SENSORS
-    _LOGGER.debug("Reading sensors configuration...")
-    if os.path.isfile(sensors_config_file):
-        SENSORS = read_yaml_file(sensors_config_file)
+    LOGGER.debug("Reading sensors configuration...")
+    if os.path.isfile(SENSORS_CONFIG_FILE):
+        SENSORS = read_yaml_file(SENSORS_CONFIG_FILE)
     else:
-        _LOGGER.info("No sensors config file found.")
+        LOGGER.info("No sensors config file found.")
 
     for sensor_mac in SENSORS:
         if valid_sensor_mac(sensor_mac):
             send_discovery_topics(sensor_mac)
 
     # Check config against linked sensors
-    result = wysesense_dongle.List()
-    _LOGGER.debug("Linked sensors: {0}".format(result))
+    result = WYSESENSE_DONGLE.List()
+    LOGGER.debug("Linked sensors: {0}".format(result))
     if result:
         for sensor_mac in result:
             if valid_sensor_mac(sensor_mac):
                 if SENSORS.get(sensor_mac) is None:
                     add_sensor_to_config(sensor_mac)
                     send_discovery_topics(sensor_mac)
     else:
-        _LOGGER.warn("Sensor list failed with result: {0}".format(result))
+        LOGGER.warning("Sensor list failed with result: {0}".format(result))
 
 # Validate sensor MAC
 def valid_sensor_mac(sensor_mac):
```

Comment:
The sensor_mac can be a valid number so len() will not work. Need something like `len(str(sensor_mac))`. One of my sensors is `77890452`

Reply:
Thank you on this one. I'm brand new to Python and not yet familiar with functions and casting requirements around variable type. I'm sure you just saved me a headache the first time I get a sensor with an all numerical mac. This is also in the next commit group.

Reply:
Just noticed you'd already done this while I was trying to recommit. So thanks! I'll resolve!